build/wikia/scrapers/ArcaneScraper.mjs

```diff
@@ -0,0 +1,8 @@
+import WikiaDataScraper from '../WikiaDataScraper.mjs';
+import transformArcanes from '../transformers/transformArcanes.mjs';
+
+export default class ArcaneScraper extends WikiaDataScraper {
+  constructor() {
+    super('https://warframe.fandom.com/wiki/Module:Arcane/data?action=edit', 'Arcane', transformArcanes);
```

Comment:
use the new warframe wiki
```suggestion
    super('https://wiki.warframe.com/w/Module:Arcane/data?action=edit', 'Arcane', transformArcanes);
```
(yes, i know mod scraper is outdated, i'm going to update shortly)